atlasdb-cassandra/src/main/java/com/palantir/atlasdb/keyvalue/cassandra/CassandraTopologyValidator.java

```diff
@@ -113,7 +81,39 @@ public Set<CassandraServer> getNewHostsWithInconsistentTopologiesAndRetry(
      * @param allHosts All Cassandra servers which must include newlyAddedHosts.
      * @return Set of Cassandra servers which do not match the pre-existing hosts topology.
```

Comment:
Let's update this too to include the change in the algo (not considering soft failures from the candidate set)